automatic/PDFCreator/update.ps1

```diff
@@ -0,0 +1,24 @@
+import-module au
+
+$url = 'http://download.pdfforge.org/download/pdfcreator/PDFCreator-stable?download'
+
+function global:au_SearchReplace {
+    @{
+        'tools\ChocolateyInstall.ps1' = @{
+            "(^[$]checksum\s*=\s*)('.*')"   = "`$1'$($Latest.Checksum32)'"
+        }
+     }
+}
+
+function global:au_GetLatest {
+    Write-Host $url
+
+    $temp_file = $env:TEMP + '\PDFCreator.exe'
+    Invoke-WebRequest $url -OutFile $temp_file
+
+    $version = (Get-Command $temp_file).Version
```

Comment:
> The suggestion wasn't to simply add a .0, but rather, if the version number is a 2 part version number, add the .0 onto it.

I think this is irrelevant. 

> ok, I have to ask, why is 2.5 not the same as 2.5.0? 

Because `[version]2.5 -ne [version]2.5.0` :)